azure-client-runtime/src/main/java/com/microsoft/azure/AzureEnvironment.java

```diff
@@ -97,6 +101,8 @@ public AzureEnvironment(Map<String, String> endpoints) {
         // TODO: add dns suffixes for the US government for datalake store and datalake analytics once they are defined.
         put("azureDataLakeStoreFileSystemEndpointSuffix", "N/A");
         put("azureDataLakeAnalyticsCatalogAndJobEndpointSuffix", "N/A");
+        put("azureLogAnalyticsResourceId", "N/A");
```

Comment:
https://api.loganalytics.us/

Need to double check if we have made App Insights available in US gov.

Reply:
Not available yet; just Log Analaytics to add here.